packages/sdk/src/authentication/bearerAuthProvider.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AxiosRequestConfig } from "axios";
+import { IAuthProvider } from "./iAuthProvider";
+
+/**
+ * Provider that handles Bearer Token authentication
+ *
+ * @beta
+ */
+export class BearerAuthProvider implements IAuthProvider {
```

Comment:
Can we rename it to `BearerTokenAuthProvider` since bearer token is a well known term

Reply:
Fixed.